plugin/cache/README.md

```diff
@@ -34,6 +34,7 @@ cache [TTL] [ZONES...] {
     success CAPACITY [TTL] [MINTTL]
     denial CAPACITY [TTL] [MINTTL]
     prefetch AMOUNT [[DURATION] [PERCENTAGE%]]
+    serve_expired yes|no [DURATION]
```

Comment:
the new metrics should also be added to the README

Reply:
Also: This be true|false instead of yes|no as far as I can see.

Reply:
I've added the metrics. renamed the option to serve_stale and got rid of the yes|no argument.
Thanks! :)